src/components/common/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -11,15 +11,15 @@
         </section>
         <ul>
           <li>
-            <v-icon v-if="hasApplicant" color="green darken-2" class="dir-valid">mdi-check</v-icon>
-            <v-icon v-else color="red">mdi-close</v-icon>
-            <!-- <v-icon>mdi-circle-small</v-icon> -->
+            <v-icon v-if="isApplicantPerson && hasApplicant" color="green darken-2" class="dir-valid">mdi-check</v-icon>
+            <v-icon v-else-if="!isApplicantOrg || !hasApplicant"  color="red">mdi-close</v-icon>
```

Comment:
Shows correct validation if more than one applicants in the List.

Reply:
Screenshot please :)

Reply:
![image](https://user-images.githubusercontent.com/116035339/225775808-ee3794d7-fe84-4411-b05d-41de78e48759.png)